static/app/views/replays/detail/network/index.tsx

```diff
@@ -72,6 +69,23 @@ function NetworkList({networkSpans, startTimestampMs}: Props) {
       deps,
     });
 
+  const initialRequestDetailsHeight = useMemo(
+    () => Math.max(150, window.innerHeight * 0.25),
```

Comment:
We may want to think about increasing the initial height in the future, I always find myself wanting more space, but could just be me.

Reply:
can do! it's a very arbitrary size right now tbh

I think also i'd want to have it scroll so that the item you clicked on is still visible after the panel opens up. right now if you click on the item at the bottom of the list the panel will jump up and cover it. i'll make a followup with these, we can tweak the default size there.